db/migrate/20210809095635_add_token_to_user_notifications.rb

```diff
@@ -0,0 +1,5 @@
+class AddTokenToUserNotifications < ActiveRecord::Migration
```

Comment:
I realised one thing -- won't the token column need to be non null? You need to add a token to all of the existing user notifications in a migration.

Reply:
is before_create will do it for ya @donjar ?

Reply:
@nugriz you haven't make the column to be non null. Can you change it? You can just alter the column in a new migration to make the column non null.